LLama.Examples/LLama.Examples.csproj

```diff
@@ -29,7 +29,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="7.0.0" />
-    <PackageReference Include="Microsoft.SemanticKernel" Version="1.0.0-beta4" />
+    <PackageReference Include="Microsoft.SemanticKernel" Version="1.0.0-beta1" />
```

Comment:
Downgrading the SemanticKernel version?

Reply:
(from `beta4` to `beta1`)

Reply:
Yes because the LLamaSharp.semantic-kernel package depends on beta1, though using beta4 seems to have no error.